modules/local/report.nf

```diff
@@ -19,6 +19,8 @@ process REPORT {
     path multiqc
     path custom_gmt
     path custom_background
+    path report_file
+    path references_file
```

Comment:
I dont see `$references_file` used?

Reply:
It isn't (or at least not in the report.nf), but it's used by the Rmd so it has to be present in the workdir of the report, that's why I added it